### PR TITLE
feature/replace-span-with-buttons

### PR DESCRIPTION
**Pull request by pawellysy**
Improved semantics of Stock Tools and popups to for better accessibility, specifically use `<button>` elements rather than spans.

TODO: 

- [x] Keep the separator in the stock tools to be a span

### CODE REVIEW

File: ts/Extensions/Annotations/Popup/PopupAnnotations.ts [hunk 1]
```diff
@@ -154,14 +154,25 @@ function addToolbar(
...
-        ))
+    let label = createElement(
+        'p',
+        {
+            className: 'highcharts-annotation-label'
+        },
+        void 0,
+        popupDiv
+    );
+    label.setAttribute('aria-label', 'Annotation type');
```
Comment (TorsteinHonsi) [inline ts/Extensions/Annotations/Popup/PopupAnnotations.ts:165]: The `createElement` function takes attributes as the second argument. It should work with the `aria-label` attribute too...

**Review by TorsteinHonsi — COMMENTED** [review]
Looks good except the inline comment, but I suspect you already tried that since you're obviously aware of the attributes argument.

File: ts/Extensions/Annotations/Popup/PopupAnnotations.ts (hunk 1 above)
Comment (pawellysy) [reply ts/Extensions/Annotations/Popup/PopupAnnotations.ts:165]: Yes, I've tried with the second argument, and something was broken. This is one point, which I have to test out, to see, how exactly can I add the `aria-label` to the correct element. Thanks!

**Review by pawelfus — CHANGES_REQUESTED** [review]
Just one thing: the mini overlay changed too much (wrong order of buttons, missing padding, wrong background colors):
Current:
Zrzut ekranu 2024-01-17 o 13 02 14
PR:
Zrzut ekranu 2024-01-17 o 13 02 20

**Review by kamil-musialowski — APPROVED** [review]

**Review by bm64 — APPROVED** [review]

**Review by pawelfus — CHANGES_REQUESTED** [review]
Almost there, just the last thing, the cursor: pointer is missing

hover

**Review by DJTechnoo — APPROVED** [review]

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! There's a failing test in the headless browser tests, that is related to Stock Tools. Can you look into that? Otherwise all good 👍

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**